python-package/src/opchatserver/server.py

```diff
@@ -47,6 +47,18 @@ def _get_origns() -> List[str]:
     return os.environ.get("ORIGINS", "http://localhost:3000").split(",")
 
 
+def _get_openai_model() -> str:
+    """
+    Get the OpenAI model that is used by the server.
+
+    Returns
+    -------
+    str
+        The OpenAI model that is used by the server.
+    """
+    return os.environ.get("OPENAI_MODEL", "gpt-3.5-turbo")
```

Comment:
If we want to make this configurable via env variable, can you add documentation on it somewhere? I also don't really see the point of making it configurable at all just for the demo, but no strong opinion

Reply:
> If we want to make this configurable via env variable, can you add documentation on it somewhere? 

Yes, will do.

>  I also don't really see the point of making it configurable at all just for the demo, but no strong opinion

I guess there is still some value that we can demo a better practice.  :)